cmd/akamai-purger/main.go

```diff
@@ -193,36 +194,54 @@ func (ap *akamaiPurger) len() int {
 
 func (ap *akamaiPurger) purge() error {
 	ap.Lock()
-	queueEntries := ap.toPurge[:]
-	ap.toPurge = [][]string{}
-	ap.Unlock()
-	if len(queueEntries) == 0 {
+	queueSize := len(ap.toPurge) - 1
```

Comment:
I think the queue _size_ is exactly the same as the queue _len_, right? Why subtract 1?

Reply:
Yeah, this was as mistake when I was trying some things out in testing. This never made it into the final commit.